### DISCUSSION TITLE
Does zephyrrtos USB CDC support Percepio Tracealyzer？

### DISCUSSION THREAD

**Discussion by hongshui3000** [zephyrproject-rtos/zephyr#42076] [Q&A] [answered]
Does zephyrrtos USB CDC_ACM support Percepio Tracealyzer？
or
Is there such a plan?

I know it seems that Percepio Tracealyzer supports stm32 cdc, but I don't know if it can be used in zephyrrtos. The code doesn't seem to work.

**Comment by tleksell-pe** [comment, marked as answer]
Hello @hongshui3000, Percepio Tracealyzer for Zephyr does not yet offer USB-CDC as a streaming interface. It is something which we are looking at and would be a good addition to Tracealyzer for Zephyr.
